sky/resources.py

```diff
@@ -267,6 +280,34 @@ def disk_size(self) -> int:
     def image_id(self) -> Optional[Dict[str, str]]:
         return self._image_id
 
+    @property
+    def disk_type(self) -> str:
+        return self._disk_type
+
+    def get_cloud_disk_type_str(self) -> str:
+        if isinstance(self.cloud, clouds.GCP):
+            return ('pd-ssd' if self.disk_type == 'high' else 'pd-balanced'
+                    if self.disk_type == 'medium' else 'pd-standard')
+        if isinstance(self.cloud, clouds.AWS):
+            return ('io2' if self.disk_type == 'high' else
+                    'gp3' if self.disk_type == 'medium' else 'gp3')
+        if isinstance(self.cloud, clouds.Azure):
+            return (
+                # TODO(tian): maybe use UltraSSD_LRS?
+                'Premium_LRS' if self.disk_type == 'high' else 'StandardSSD_LRS'
+                if self.disk_type == 'medium' else 'Standard_LRS')
+        raise NotImplementedError(
+            f'Disk type of cloud {self.cloud} is not implemented yet.')
+
+    def get_disk_iops(self) -> int:
+        if isinstance(self.cloud, clouds.AWS):
+            # TODO(tian): benchmark on each cloud to make sure performance
+            # is similar
+            return (27000 if self.disk_type == 'high' else
+                    9000 if self.disk_type == 'medium' else 3000)
+        raise NotImplementedError(
+            f'Disk type of cloud {self.cloud} is not implemented yet.')
```

Comment:
If we move the configuration into `cloud.make_deploy_resources_variables()` as mentioned above, so we don't have to do this here.